express/gcp_storage.py

```diff
@@ -103,23 +103,13 @@ def copy_files(source_files: List[str], destination: str):
              copy
             destination (str): the destination blob/folder to copy the files to
         """
+        with subprocess.Popen(  # nosec
+            ['gsutil', '-o', '"GSUtil:use_gcloud_storage=True"', '-q', '-m', 'cp', '-I',
+             destination], stdin=subprocess.PIPE
+        ) as gsutil_copy:
+            gsutil_copy.communicate(b"\n".join([f.encode() for f in source_files]))
 
-        # Write file paths to a text file before piping
```

Comment:
This code threw bandit warnings, so I rewrote it as a shorter piece of code that still throws bandit warnings and suppressed them :sweat_smile: